targets/arm64.json

```diff
@@ -0,0 +1,24 @@
+{
+	"build-tags": ["arm64"],
+	"goarch":"arm64",
+	"goos":"linux",
+	"llvm-target":"aarch64-elf",
+	"tags":"arm64",
+	"compiler":"clang",
+	"linker":"ld.lld",
+	"rtlib": "compiler-rt",
+	"libc": "picolibc",
```

Comment:
I think it's best to avoid picolibc for now. I would like to first see why Clang doesn't use the usual `<sysroot>/include` but instead uses `<sysroot>/usr/include`, it doesn't make sense that it only does this for aarch64 (and not for arm, riscv, and avr) and I'm hopeful there is some sort of flag to tell it not to do this. In any case, those changes can be done in a separate PR (a libc should not be necessary for initial bringup of this target).

Reply:
Removed.